manifest_staging/charts/workload-identity-webhook/README.md

```diff
@@ -31,7 +31,6 @@ helm upgrade -n azure-workload-identity-system [RELEASE_NAME] azure-workload-ide
 
 | Parameter                     | Description                                                                                                                                                          | Default                                                 |
 | :---------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------ |
-| labels                        | The labels to add to the azure-workload-identity webhook pods                                                                                                        | `azure-workload-identity.io/system: "true"`             |
```

Comment:
`labels` was never used in the chart, so it should be safe to rename it to `podLabels` to be more specific